climpred/relative_entropy.py

```diff
@@ -0,0 +1,301 @@
+import warnings
+
+import numpy as np
+import xarray as xr
+from eofs.xarray import Eof
+
+
+def _relative_entropy_formula(sigma_b, sigma_x, mu_x, mu_b, neofs):
```

Comment:
This should be a sub-function of `compute_relative_entropy` since it is only called there.

Reply:
I agree in theory, but then how can the user look into that function, ie. see the documentation and references therein?

Reply:
If it's an underscore function, then they shouldn't be accessing it directly. You would put the documentation of this formula into the documentation of the parent function. I can do this when I squash the commits.